src/components/v2/Table/styles.ts

```diff
@@ -29,7 +28,12 @@ export const useStyles = () => {
 
       .MuiTableCell-root:first-child {
         padding-left: ${theme.spacing(6)};
-        float: left;
+        text-align: left;
+        max-width: 110px;
```

Comment:
Two things:
- we've agreed to use `theme.spacing` to express values like this
- we shouldn't have hard-coded values like this in my opinion. At the moment I'm guessing you're updating this component so it displays well on the dashboard, however that component will be reused on other screens that won't use the same columns. We should think of a way to make this component reusable

Reply:
added gridTemplateColumns prop that handles columns width.